runner/src/main/resources/logback.xml

```diff
@@ -14,5 +14,7 @@
     <root level="debug">
         <appender-ref ref="stdout"/>
     </root>
-    <logger name="io.netty" level="info"/>
+    <logger name="org.jboss.logging" level="off"/>
```

Comment:
The `org-jboss` could be used here to prevent the message that comes in upon launching the applciation (unless it was intentional):

```suggestion
    <logger name="org.jboss" level="off"/>
```